dotCMS/src/main/java/com/dotmarketing/servlets/LoginEditModeServlet.java

```diff
@@ -77,5 +84,14 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
 		}
 		request.getRequestDispatcher("/c").forward(request,response);
 	}
+
+    private String findAssetURI(Contentlet cont) throws DotStateException, DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)